src/main/java/core/basesyntax/Information.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Information {
```

Comment:
Let's change the naming. The interface often ends with "-able". E.g. `Serializable, Closable` etc. So, let's use `Drawable`.